src/mruby-zest/example/ZynReverse.qml

```diff
@@ -0,0 +1,60 @@
+Group {
+    id: reverse
+    label: "reverse"
+    topSize: 0.2
+    function refresh() {
+        return if rw.content.nil?
+        return if rw.content.children.length < 4
+        rw.content.children[4..-1].each do |c|
+            c.refresh
+        end
+    }
+
+    ParModuleRow {
+        id: rw
+        layoutOpts: []
+        Selector {
+            extern: reverse.extern + "Reverse/preset"
+            whenValue: lambda { reverse.refresh }
+        }
+        Knob { extern: reverse.extern + "Pvolume"}
+        Knob { extern: reverse.extern + "Ppanning"}
+        Col {
+            NumEntry {extern: reverse.extern + "numerator"; 
+                id: num
+                value: 0
+                label: "Numerator"
+                whenValue: lambda { reverse.refresh } 
+            }
+            NumEntry {extern: reverse.extern + "denominator"; 
+                id: den
+                value: 4
+                label: "Denominator"
+                whenValue: lambda { reverse.refresh }
+            }
+        }
+        Knob { 
```

Comment:
I cannot see the controls for delay, crossfade and Syncmode. Maybe you need to update test.json?

Reply:
I forgot `git pull` on the UI, now it works. Resolving my comment.